From 321003077000ed32adf15f0d11d2b533a71ef519 Mon Sep 17 00:00:00 2001
From: Jon Medhurst <tixy@linaro.org>
Date: Wed, 1 Feb 2012 15:56:32 +0000
Subject: [PATCH 3345/3348] vexpress-hacks: Update default bootargs to set root=/dev/mmcblk0p2

This means they can boot Ubuntu images out-of-the-box as well as Android.

Signed-off-by: Jon Medhurst <tixy@linaro.org>
---
 include/configs/vexpress_common.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/configs/vexpress_common.h b/include/configs/vexpress_common.h
index e6ea244..0c3a07e 100644
--- a/include/configs/vexpress_common.h
+++ b/include/configs/vexpress_common.h
@@ -262,7 +262,7 @@
 		"maxramdisk=0x1800000\0" \
 		"pxefile_addr_r=0xa8000000\0" \
 		"kernel_addr_r=0xa0008000\0" \
-		"mmc_args=setenv bootargs \"console=tty0 console=ttyAMA0,38400n8 rootwait ro mmci.fmax="CONFIG_MMCI_FMAX"\"; "
+		"mmc_args=setenv bootargs \"console=tty0 console=ttyAMA0,38400n8 root=/dev/mmcblk0p2 rootwait ro mmci.fmax="CONFIG_MMCI_FMAX"\"; "
 #endif
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
-- 
1.7.0.1

